examples/nodejs/express-ts/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+---
+version: '3.9'
+services:
+  pyroscope:
+    image: 'pyroscope/pyroscope:latest'
+    ports:
+      - '4040:4040'
+    command:
+      - 'server'
+  app:
+    build: .
+    cap_add:
+      - SYS_PTRACE
```

Comment:
Is this actually needed or just bad copy paste?

Reply:
ofc copy-paste )